_i18n/de/resources/user-guides/mine-to-pool.md

```diff
@@ -151,94 +127,70 @@ After these modifications, your configuration should look something like this:
-to infected computers to mine without the owner's consent. As it is
-your computer and you are configuring the miner to mine for you, it is
-safe to add XMRig to your anti-virus whitelist.
+Manche Antivirenprogramme markieren XMRig als Malware, weil es häufig auf "infizierten" Rechnern installiert wird, um dadurch ohne das Einverständnis des Besitzers mit diesen Computern minen zu können. Da es dein eigener Rechner ist und du den Miner dazu einrichtest, für dich selbst zu minen, kannst du XMRig sicher auf die Whitelist deines Antivirenprogramms setzen. 
 
-### Cannot read/set MSR
+### MSR kann nicht gelesen/eingerichtet werden
 
-On some CPUs, XMRig tries to increase performance by disabling certain
-features like your CPU's instruction prefetcher. These operations
-require root/administrator, so try right clicking xmrig.exe and
-running it as administrator, or running `sudo ./xmrig` on other
-systems.
+Bei manchen CPUs versucht XMRig es, die Leistung durch Deaktivierung bestimmter Funktionen (etwa des Anweisungsprefetchers deines CPUs) zu verbessern. Diese Vorgänge benötigen Administratorrechte beziehungsweise den Gebrauch des Root-Kontos. Versuche, es mit einem Rechtsklick auf xmrig.exe als Administrator auszuführen oder es auf anderen Systemen durch den Befehl `sudo ./xmrig` zu starten.
 
-### Algo not known
+### Algo nicht bekannt
 
-Find the line in config.json that says `algo: null` and change it to
-`algo: "rx/0"`. By default, XMRig expects the pool to tell it which
-hashing algorithm to use.
+Suche in der "config.json"-Datei nach der Zeile, die `algo: null` enthält, und ändere dies zu `algo: "rx/0"`. Standardmäßig wartet XMRig nun darauf, dass der Pool mitteilt, welcher Hashing-Algorithmus verwendet werden soll.
 
-### Huge Pages 0%
+### Große Seiten 0%
 
-#### Allowing large pages on Windows
+#### Große Seiten auf Windows erlauben
 
-Taken from [the MSDN](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows?view=sql-server-ver15):
+Von [MDSN](https://docs.microsoft.com/de-de/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows?view=sql-server-ver15) übernommen:
 
->1. On the Start menu, click Run. In the Open box, type gpedit.msc.
-2. On the Local Group Policy Editor console, expand Computer Configuration, and then expand Windows Settings.
-3. Expand Security Settings, and then expand Local Policies.
-4. Select the User Rights Assignment folder.
-5. The policies will be displayed in the details pane.
-6. In the pane, double-click Lock pages in memory.
-7. In the Local Security Setting – Lock pages in memory dialog box, click Add User or Group.
-8. In the Select Users, Service Accounts, or Groups dialog box, add an account that you will run the miner on
-9. Reboot for change to take effect.
+>1. Klicken Sie im Menü "Start" auf "Ausführen". Geben Sie "gpedit.msc" im Feld "Öffnen" ein.
```

Comment:
I noticed that you switched to "Sie" instead of "du" in this part. Most other parts seem to use "du".

Reply:
Also taken from the MDSN and can be adjusted accordingly (again with square brackets) if we decide on it.